R/loadWorkbook.R

```diff
@@ -7,6 +7,7 @@
 #' @param file A path to an existing .xlsx or .xlsm file
 #' @param xlsxFile alias for file
 #' @param isUnzipped Set to TRUE if the xlsx file is already unzipped
+#' @param na.convert Should empty/blank cells be converted to `NA_character_`. Defaults to TRUE.
```

Comment:
this needs a roxygen run.

Reply:
Not sure what that means - that's a bit beyond what I can contribute I'm afraid.

Reply:
You simply have to run roxygen2 to create the man pages, not sure what the exact command is. In Rstudio it's Ctrl+shift+d. This will update the man pages.

Reply:
I see, I think it did that automatically and this shoudl be added in my second commit. At least I saw the change to the man page there.

Reply:
ah yes, I was fooled by your comment :)